packages/playwright-test/src/plugins/vitePlugin.ts

```diff
@@ -69,7 +69,7 @@ export function createPlugin(
           },
         },
         sourcemap: true,
-        outDir: viteConfig?.build?.outDir || path.join(viteConfig.root, './dist-pw/')
+        outDir: viteConfig?.build?.outDir || path.join(viteConfig.root, 'playwright', 'out')
```

Comment:
Should we add this to the create-playwright defaults?
(Maybe already done)

Reply:
It is done, yes